pulp_rpm/tests/functional/constants.py

```diff
@@ -313,6 +313,12 @@
 SRPM_UNSIGNED_FIXTURE_URL = urljoin(PULP_FIXTURES_BASE_URL, 'srpm-unsigned/')
 """The URL to a repository with unsigned SRPM packages."""
 
+SRPM_UNSIGNED_FIXTURE_PACKAGE_COUNT = 3
+"""Count of packages in the repository."""
+
+SRPM_UNSIGNED_FIXTURE_ADVISORY_COUNT = 2
```

Comment:
I know I'm a pain :) This should be before package count variable if we follow alphabetical order.

Reply:
it is good to keep the code in shape, moved.